src/marqo/enums.py

```diff
@@ -16,3 +16,4 @@ class IndexStatus:
     DELETED = "DELETED"
     MODIFYING = "MODIFYING"
     CREATING = "CREATING"
+    DELETING = "DELETING"
```

Comment:
What is the possible impact of this? I am nervous this could introduce a regression...